lib/ansible/modules/cloud/ovirt/ovirt_storage_domains.py

```diff
@@ -115,6 +115,22 @@
             - "C(port) - Port of the fibre channel storage server."
             - "C(lun_id) - LUN id."
             - "Note that these parameters are not idempotent."
+    wipe_after_delete:
+        description:
+            - "Boolean flag which indicates whether the storage domain should wipe the data after delete."
+        version_added: "2.5"
+    backup:
+        description:
+            - "Boolean flag which indicates whether the storage domain is configured as backup or not."
+        version_added: "2.5"
+    critical_space_action_blocker:
+        description:
+            - "Inidcates the minimal free space the storage domain should contain in GB percentages."
```

Comment:
So it's in gigabytes or in percentage, it's not clear from the description.

Reply:
My bad, it is in percentage, will fix